backend/app/main_test.go

```diff
@@ -158,5 +158,6 @@ func TestMain(m *testing.M) {
 		m,
 		goleak.IgnoreTopFunction("github.com/umputun/remark42/backend/app.init.0.func1"),
 		goleak.IgnoreTopFunction("net/http.(*Server).Shutdown"),
+		goleak.IgnoreTopFunction("github.com/blevesearch/bleve_index_api.AnalysisWorker"),
```

Comment:
probably there is a reason for this ignore, but it will be great to add a comment explaining why it ign ored

Reply:
Added comment. Actually, we can also shutdown this worker by calling `belve.Config.Shutdown()`, but it's a global worker, and it would be stopped forever. So, I suppose we can leave it as is, and killing these goroutines in `main` doesn't makes much sense.